### ISSUE TITLE
Include link to quaddicted page

### ISSUE THREAD

**Issue by WRFitch** [hrehfeld/QuakeInjector#135]
Since #66, #69, and #76 all want to re-implement functionality that already exists on quaddicted, why not include a link to the page to make these functions easier in the short term? This also makes it possible for people to use browser favourites to keep a list of maps they prefer, and it makes it faster to write a review of a map you've just played/tested.

**Comment by SpiritQuaddicted** [comment]
I'd love that! Not sure why we don't have it already :)

**Comment by SpiritQuaddicted** [comment]
Turns out you can click the packages title (above the screenshot) and it will open the user's browser on the review page already.

**Comment by SpiritQuaddicted** [comment]
People in Discord (and myself) think it would be best to have a button for it.

**Comment by WRFitch** [comment]
I actually like clicking the title since it stops the interface being too cluttered, but I agree it could be telegraphed better. I'll have a look at implementing this sometime next week

**Comment by SpiritQuaddicted** [comment]
Agreed! Maybe just underlining it and using a slightly different color would work.

**Comment by WRFitch** [comment]
I thought about making it a more obvious hyperlink, but I'm not sure it would look right. I'll experiment with a dropdown menu or different button configurations at some point and get back to you, work is taking up all my time at the moment :)

**Comment by ericwa** [comment]
maybe a "globe" icon, or "chain link" icon?

**Comment by johnnycz** [comment]
I believe this is resolved in version 7 as we list all links there explicitly. So this could be closed?

**Comment by SpiritQuaddicted** [comment]
Oh yeah, thank you! 🥳
